packages/mui-material/src/Accordion/Accordion.d.ts

```diff
@@ -32,6 +32,7 @@ export interface AccordionProps extends StandardProps<PaperProps, 'onChange'> {
   /**
    * If `true`, expands the accordion, otherwise collapse it.
    * Setting this prop enables control over the accordion.
+   * @default undefined
```

Comment:
We should not use `undefined` as the default value in the JS Docs. If a property does not have any default value, we can consider that it is undefined, and we should not document it as having any value. I would look in more detail at the logic in `ComponentApiBuilder` instead.